src/components/BookmarkPage.js

```diff
@@ -3,16 +3,47 @@ import RestaurantBox from "./RestaurantBox";
 import { ReactComponent as Find } from "../icons/find.svg";
 import { Link } from "react-router-dom";
 
-export default function BookmarkPage() {
+export default function BookmarkPage({ restaurantData }) {
+  const bookmarkedRestaurants = restaurantData.filter(
+    (savedRestaurants) => savedRestaurants.isBookmarked === true
+  );
+
+  function renderRestaurants() {
+    return bookmarkedRestaurants.map((restaurant) => {
+      const {
+        id,
+        name,
+        kitchen,
+        location,
+        atmosphere,
+        isBookmarked,
+      } = restaurant;
+      const cuisine = kitchen[0];
+      const area = location.area[0];
+      const ambience = atmosphere[0];
```

Comment:
Now that we are talking about destructuring, you can achieve the same here with this other syntax:
```js
const [cuisine] = kitchen; // equivalent to const cuisine = kitchen[0];
```

Just another way.
This is what we do with `useState`. `useState` returns an array with 2 elements, and we destructure it always because it's easier to read.